docs/content/developer/getting-started/get-coins.mdx

```diff
@@ -36,10 +37,10 @@ curl --location --request POST 'https://faucet.testnet.iota.cafe/gas' \
 
 :::tip Test tokens on a local network
 
-If you're working with a local network, replace `'https://faucet.devnet.iota.cafe/gas'` with the appropriate value based on which package runs your network:
+If you're working with a local network, replace <>{Networks.iota_move_devnet.faucetUrl}</> with the appropriate value based on which package runs your network:
 
 - `iota-faucet`: `http://127.0.0.1:5003/gas`
-- `iota start`: `http://127.0.0.1:9123/gas`
+- `iota start`: <code>{Networks.iota_localnet.faucetUrl}</code>
```

Comment:
Not really related to this PR, but this is written in a weird way if you ask me. Can we change this admonition?
We can also look at it in another PR

Reply:
Ok, I will update in the next commit.

Reply:
Seems like you removed the const here. Not sure why. I also asked for clarification on slack for this part. But we can merge your PR in the meantime. Let's just get the other last things resolved 👍